### PR TITLE
Add op-level test YAML files for Gemma 4 26B

### PR DESCRIPTION
**Pull request by lipikaworkemail-ctrl**
Thanks for sending a pull request! Please make sure you read the [contributing guidelines](https://github.com/torch-spyre/torch-spyre/blob/main/CONTRIBUTING.md).

#### What type of PR is this?

- [ ] bug
- [X] feature
- [X] documentation
- [ ] cleanup

#### What this PR does:

This PR adds YAML files for the `google/gemma-4-26B-A4B-it` model to support model op-level test cases. This model is currently a high priority, so adding these YAMLs will help enable and accelerate its testing.
These YAML files were generated using the op tracing framework described in [model-optest-yaml-generator](https://github.ibm.com/ai-foundation/aiu-tests/tree/model-opstest-yaml-generator/utils/model-opstest-yaml-generator).

The 26B model is a Mixture-of-Experts (MoE) variant of the Gemma 4 family. It shares the base transformer architecture with the 12B model (see PR #3178) but introduces an additional MoE feed-forward block, resulting in 14 new ops relative to the 12B op set.

#### Does this PR introduce a user-facing change?
No. The generator script and YAML files are internal tooling only.

#### Op list (49 ops):
```
torch.Tensor.contiguous
torch.Tensor.expand
torch.Tensor.mul
torch.Tensor.truediv
torch.Tensor.view
torch.add
torch.add_
torch.arange
torch.cat
torch.chunk
torch.clamp_
torch.clone
torch.cos
torch.cumsum
torch.empty_like
torch.eq
torch.float
torch.floordiv
torch.ge
torch.getitem
torch.histc
torch.index_copy_
torch.masked_fill_
torch.matmul
torch.mean
torch.mul
torch.neg
torch.nn.functional.embedding
torch.nn.functional.gelu
torch.nn.functional.linear
torch.nn.functional.scaled_dot_product_attention
torch.nn.functional.softmax
torch.ops.transformers.grouped_mm_fallback
torch.or_
torch.pow
torch.reshape
torch.setitem
torch.sin
torch.sort
torch.sum
torch.tanh
torch.to
torch.topk
torch.transpose
torch.truediv
torch.type_as
torch.unsqueeze
torch.where
torch.zeros

```

### CODE REVIEW

**Comment by lipikaworkemail-ctrl** [comment]
cc: @ani300 @saurabh-srivastava4 @goutham-binnadi-gopala

File: tests/configs/model_ops_tests/gemma-4-26B-A4B-it.yaml [hunk 1]
```diff
@@ -0,0 +1,4852 @@
+test_suite_config:
```
Comment (anubhavjana) [inline tests/configs/model_ops_tests/gemma-4-26B-A4B-it.yaml:1]: Please add `labels` like https://github.com/torch-spyre/torch-spyre/blob/599144f9d32de59d36c5d8e49f2b7b3c045703c7/tests/configs/model_ops_tests/gemma-4-12B-it.yaml#L2

File: tests/configs/model_ops_tests/gemma-4-26B-A4B-it_spyre.yaml [hunk 2]
```diff
@@ -0,0 +1,4641 @@
+test_suite_config:
+  global:
```
Comment (anubhavjana) [inline tests/configs/model_ops_tests/gemma-4-26B-A4B-it_spyre.yaml:3]: Please add `labels` like https://github.com/torch-spyre/torch-spyre/blob/599144f9d32de59d36c5d8e49f2b7b3c045703c7/tests/configs/model_ops_tests/gemma-4-12B-it.yaml#L2

**Review by anubhavjana — CHANGES_REQUESTED** [review]

**Comment by kiszk** [comment]
Could you apply [this fix](https://github.com/torch-spyre/torch-spyre/pull/3773) since the generator seems to have a bug?

**Comment by kiszk** [comment]
Could you apply [this change](https://github.com/torch-spyre/torch-spyre/pull/3723)?

**Comment by kiszk** [comment]
Could you please attach the log of executing tests for v1 and v2?

**Comment by kiszk** [comment]
cc @moriohara

File: tests/configs/model_ops_tests/gemma-4-26B-A4B-it.yaml (hunk 1 above)
Comment (lipikaworkemail-ctrl) [reply tests/configs/model_ops_tests/gemma-4-26B-A4B-it.yaml:1]: Added `labels`

File: tests/configs/model_ops_tests/gemma-4-26B-A4B-it_spyre.yaml (hunk 2 above)
Comment (lipikaworkemail-ctrl) [reply tests/configs/model_ops_tests/gemma-4-26B-A4B-it_spyre.yaml:3]: Added `labels`

**Comment by lipikaworkemail-ctrl** [comment]
Thanks for the review @anubhavjana and @kiszk.
Applied all suggested fixes. Logs from running the test YAMLs in both v1 (old format) and v2 (new format) are attached.
1. Old format: [gemma26_log_v1_spyre.txt](https://github.com/user-attachments/files/31149082/gemma26_log_v1_spyre.txt)
2. New format: [gemma26_log_v2_spyre.txt](https://github.com/user-attachments/files/31149088/gemma26_log_v2_spyre.txt)

**Review by anubhavjana — APPROVED** [review]
LGTM
